### ISSUE TITLE
To-do before February 22nd

### ISSUE THREAD

**Issue by katieb1** [katieb1/NEvolve#1]
- [x] Write up project background in README.md
- [x] Get started on [cedar](https://gist.github.com/razoumov/15f54616b62269a3fa05fbc8cba4b92e)
- [ ] NSight tutorial + load on cluster
- [ ] Get acquainted with GPUs (see slide deck from Feb 12th presentation for tutorial links)
- [ ] Complete a training run with the single demographic scenario data through the hyperparameter tuning script and CNN script
- [ ] Get acquainted with multithreading in PyTorch for reading and writing data (`pytorch.multiprocessing`, pre-fetching, and queuing)
- [ ]  Write data to hdf5 format so it is batch-ready (demographic scenarios should be normalized and randomized across batches, and batches should be 100MB) - make sure to set random seed
- [ ] Things to think about:
* Use GPU when writing to hdf5 files
* When validation testing, accuracy metric should be weighted by class size (e.g. if you have 50 samples from a demographic scenario and then CNN gets 3 correct, then accuracy should take into account the relative sample size)
